bin/setup-local-env.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Exit if any command fails
+set -e
+
+# Change to the expected directory
+cd "$(dirname "$0")"
+cd ../docker
+
+# Launch the WordPress docker
+docker-compose up -d
+
+# Find a way to sleep until the docker containers are setup properely
+sleep 20
```

Comment:
you should look at `docker-compose ps` and `grep`:

Something like (in a loop):

```
docker-compose ps wordpress | grep 'Up'
```

Reply:
Or a simple `curl` in a loop on the WP container until the homepage responds with a `200 Ok`